src/components/LandingPage/LandingPage.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { LandingPage } from './LandingPage';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+const meta: Meta<typeof LandingPage> = {
+  title: 'Components/LandingPage',
+  component: LandingPage,
+  parameters: {
+    layout: 'fullscreen',
+    docs: {
+      description: {
+        component:
+          'Página de inicio inspirada en LeetCode con diseño moderno y gradientes atractivos.',
+      },
+    },
+  },
+} as Meta;
+
+export default meta;
+type Story = StoryObj<typeof LandingPage>;
+
+export const Default: Story = {
+  args: {},
+  parameters: {
+    docs: {
+      description: {
+        story: 'Vista por defecto de la página de landing con todos los elementos principales.',
+      },
+    },
```

Comment:
Story `Default` uses a Spanish description for the story body. Keep documentation and story comments in English and avoid hardcoded localized strings.

**Sugerencia:**
```typescript
Translate the story descriptions to English or use the project's i18n system if Storybook must show localized text.
```